via/views/__init__.py

```diff
@@ -21,10 +21,10 @@ def add_routes(config):  # pragma: no cover
         factory=QueryURLResource,
     )
     config.add_route(
-        "proxy_onedrive_pdf",
-        "/onedrive/proxied.pdf",
-        factory=QueryURLResource,
+        "proxy_onedrive_pdf", "/onedrive/proxied.pdf", factory=QueryURLResource
     )
+    config.add_route("proxy_d2l_pdf", "/d2l/proxied.pdf", factory=QueryURLResource)
+    config.add_route("proxy_python_pdf", "proxied.pdf", factory=QueryURLResource)
```

Comment:
This last route is a bit extra and not used but I reckon it leads to a nice "just proxy these with python" feature in the future without too much noise.